### ISSUE TITLE
Registration Broken?

### ISSUE THREAD

**Issue by puppycodes** [datproject/datBase#629]
When attempting to register a new account here: https://datbase.org/register
```
Uncaught DOMException: Failed to execute 'measure' on 'Performance': The mark 'end-2746000-choo.use' does not exist.
    at https://datbase.org/public/js/app.min.bd980c7098a61c6803c9b251f21dee02.js:633:374344
    at https://datbase.org/public/js/app.min.bd980c7098a61c6803c9b251f21dee02.js:633:375106
3api/v1/register Failed to load resource: the server responded with a status of 400 ()
```
Chrome Version 65.0.3325.181 (Official Build) (64-bit)

**Comment by joehand** [comment]
Hmmm. That error shouldn't affect registration. I just tried in Chrome and it worked. Are you still having issues?

**Comment by puppycodes** [comment]
Yes i'm still unable to register... the only thing I can think of is perhaps registration doesn't recognize some TLD's? My email i'm trying to register is woof@puppy.codes

**Comment by puppycodes** [comment]
if its the TLD's maybe we could add a verbose error to the submission form

**Comment by joehand** [comment]
Ya, someone else was having issues too. We definitely need to show what errors are happening to the form! Related: #630 

puppy.codes is a perfectly awesome domain, so it should definitely accept it.

**Comment by puppycodes** [comment]
haha thanks Joe! ☺️

**Comment by joehand** [comment]
Added the error message to the form. Let us know what error you're getting and hopefully we can get it fixed. Thanks!

**Comment by puppycodes** [comment]
hehe so...
<img width="360" alt="screen shot 2018-04-24 at 4 35 27 pm" src="https://user-images.githubusercontent.com/5248271/39219168-b1b7b98c-47dd-11e8-8365-edf291cab7d5.png">
<img width="366" alt="screen shot 2018-04-24 at 4 35 44 pm" src="https://user-images.githubusercontent.com/5248271/39219169-b1db749e-47dd-11e8-9d8f-ca5030d7327e.png">

**Comment by joehand** [comment]
LOL. Well, thats confusing! 

Looks like there is an account for you: https://datbase.org/puppycodes. Do you recognize what email that gravatar comes from? I think we can assume its an error with the login form now?

**Comment by karissa** [comment]
@joehand I think this happens when township adds a user successfully, but the database doesn't (or vice versa)

**Comment by puppycodes** [comment]
@joehand cool! i figured out the email connected to the account and I reset my password, so it seems just we need to differentiate between "this username is already taken" vs "this email is already registered" in terms of errors.

Thanks Again! <3

**Comment by puppycodes** [comment]
Hi, so this is still an issue strangely i can't login with my email. I'm still getting the user does not exist error, but i am able to reset my password without being able to log in after changing it or before.

**Comment by puppycodes** [comment]
@joehand nope I get "user does not exist" in the cli

**Comment by karissa** [comment]
@puppycodes did you resolve the issue?

**Comment by puppycodes** [comment]
@karissa I realized that my original assumption was probably the issue... an error semantics problem. Maybe it should say something like "username already exists". Strangely i was able to reset my password for the email that wasn't associated with any account (this is still a mystery). But it looks like my username was tied to a different email. I was able to login with that email address and change it to the one I wanted to use. Thanks for your help!
